extensions/smallrye-stork/runtime/src/main/java/io/quarkus/stork/StorkServiceDiscoveryConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.stork;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class StorkServiceDiscoveryConfiguration {
+
+    /**
+     * Configures the service discovery type, e.g. "consul".
+     * ServiceDiscoveryProvider for the type has to be available
+     *
+     */
+    @ConfigItem
+    public String type;
+
+    /**
+     * ServiceDiscovery parameters.
```

Comment:
maybe it would be good to add sth like "Check the documentation of the selected service discovery type for available parameters"?

Reply:
yes, you're right, this is a good idea, thx